opentelemetry-sdk/src/metrics/internal/histogram.rs

```diff
@@ -32,15 +31,10 @@ impl<T: Number> AtomicTracker<T> for HistogramTracker<T> {
         buckets.bin(index, value);
         buckets.sum(value);
     }
-}
-
-impl<T: Number> AtomicallyUpdate<T> for HistogramTracker<T> {
-    type AtomicTracker = HistogramTracker<T>;
 
-    fn new_atomic_tracker(buckets_count: Option<usize>) -> Self::AtomicTracker {
-        let count = buckets_count.unwrap();
+    fn create(count: &usize) -> Self {
         HistogramTracker {
-            buckets: Mutex::new(Buckets::<T>::new(count)),
+            buckets: Mutex::new(Buckets::<T>::new(*count)),
```

Comment:
Is the dereference(*) necessary as `*count`?

Reply:
yes, because function signature is `fn create(count: &usize) ...`